web/packages/teleport/src/UnifiedResources/FilterPanel.tsx

```diff
@@ -136,6 +136,7 @@ const SortOrderButton = styled(ButtonBorder)`
   border-bottom-left-radius: 0;
   border-color: ${props => props.theme.colors.spotBackground[1]};
   border-left: none;
+  height: 40px;
```

Comment:
This is fine for now. FYI, the filters are going to be changing completely soon but this will hold us over.